pyproject.toml

```diff
@@ -94,6 +94,7 @@ debugpy = "^1.8.7"
 pytest-xdist = { version = "^3.5.0", extras = ["psutil"] }
 types-requests = "^2.32.0.20241016"
 types-aiofiles = "^24.1.0.20240626"
+uvloop = "^0.21.0"
```

Comment:
Shouldn't `uvloop` be a regular dependency? Because when we run the server we run with uvloop?

Reply:
well it's really a depenceny of sanic at runtime, but in tests we call it directly to make sure everything runs in the same loop. We don't use uvloop directly in application code as far as I'm aware, so I thought it best to have it as a dependency for tests